extern/agg24-svn/include/agg_array.h

```diff
@@ -157,7 +157,7 @@ namespace agg
 
     //--------------------------------------------------------------pod_vector
     // A simple class template to store Plain Old Data, a vector
-    // of a fixed size. The data is continous in memory
+    // of a fixed size. The data is continuous in memory
```

Comment:
`extern/*` contains vendored external code. I think we want as little changes in there as possible, an thus recommend not to fix spelling in there. But maybe @tacawell and/or @anntzer should give their opinion here.

Reply:
FWIW my preference is to not touch anything in extern/